src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -17,6 +21,45 @@
  * Contains utility methods for populating {@code AddressBook} with sample data.
  */
 public class SampleDataUtil {
+
+    public static DietPlanList getSampleDietPlanList() {
+        DietPlanList list = new DietPlanList();
+        list.getPlanList().addAll(getSampleDietPlans());
+        return list;
+    }
+
+    public static DietPlan[] getSampleDietPlans() {
+        return new DietPlan[] {
+            new DietPlan("Standard Ketogenic Diet",
+            "The Standard Ketogenic Diet is a high-fat, low-carb weight-loss diet. "
+                    + "It is designed in such a way that by reducing the intake of carbohydrates, "
+                    + "the body is forced to burn its fat reserves for fuel thereby resulting in weight-loss. \n"
+                    + "\n"
+                    + "The Standard Ketogenic Diet is suitable for individuals suffering from Type II Diabetes where "
+                    + "excess carbohydrates would have been converted into glucose.\n",
+                new MacroNutrientComposition(70, 10, 20),
+                PlanType.WEIGHTLOSS),
+            new DietPlan("High-Protein Ketogenic Diet",
+                "The High-Protein Ketogenic Diet is a variation of the Ketogenic Diet (high-fat, low-carb) "
+                        + "which increases the protein intake. This variation is designed to help athletes and "
+                        + "bodybuilders maintain their muscle mass whilst burning fat.",
+                new MacroNutrientComposition(60, 35, 5),
+                PlanType.WEIGHTLOSS),
+            new DietPlan("Skinny Guy BodyBuilding",
+                "The goal for bodybuilders is to increase muscle mass. Consume high-quality, nutrient-dense "
+                        + "carbs when the body needs them most, around workouts. The hyperenergetic diet "
+                        + "plan recommends consuming starchy food during and after workouts, and less "
+                        + "starchy content on off-hours.",
+                new MacroNutrientComposition(15, 55, 30),
+                PlanType.WEIGHTGAIN),
+            new DietPlan("Balanced Plan",
+                "The perfect ying-yang. Eat healthy food and complete the calorie goal. Eat lots of fruits "
+                        + "and vegetables, and base meals on higher fiber starchy carbohydrates.",
+                new MacroNutrientComposition(30, 40, 30),
+                PlanType.WEIGHTMAINTAIN)
```

Comment:
Thanks for styling the diet plans nicely! +1.

Reply:
Ayeee! You're welcome! :DDDDD